docs/word/word-add-ins-events.md

```diff
@@ -11,37 +11,37 @@ This article describes important concepts related to working with events in Word
-| `onDataChanged` | Occurs when data within the content control are changed. To get the new text, load this content control in the handler. To get the old text, do not load it.<br><br>Event data object:<br>[ContentControlDataChangedEventArgs](/javascript/api/word/word.contentcontroldatachangedeventargs) | [**ContentControl**](/javascript/api/word/word.contentcontrol#word-word-contentcontrol-ondatachanged-member) |
-| `onDeleted` | Occurs when the content control is deleted. Do not load this content control in the handler, otherwise you won't be able to get its original properties.<br><br>Event data object:<br>[ContentControlDeletedEventArgs](/javascript/api/word/word.contentcontroldeletedeventargs) | [**ContentControl**](/javascript/api/word/word.contentcontrol#word-word-contentcontrol-ondeleted-member) |
-| `onEntered` | Occurs when the content control is entered.<br><br>Event data object:<br>[ContentControlEnteredEventArgs](/javascript/api/word/word.contentcontrolenteredeventargs) | [**ContentControl**](/javascript/api/word/word.contentcontrol#word-word-contentcontrol-onentered-member) |
-| `onExited` | Occurs when the content control is exited, for example, when the cursor leaves the content control.<br><br>Event data object:<br>[ContentControlExitedEventArgs](/javascript/api/word/word.contentcontrolexitedeventargs) | [**ContentControl**](/javascript/api/word/word.contentcontrol#word-word-contentcontrol-onexited-member) |
-| `onParagraphAdded` | Occurs when the user adds new paragraphs.<br><br>Event data object:<br>[ParagraphAddedEventArgs](/javascript/api/word/word.paragraphaddedeventargs) | [**Document**](/javascript/api/word/word.document#word-word-document-onparagraphadded-member) |
-| `onParagraphChanged` | Occurs when the user changes paragraphs.<br><br>Event data object:<br>[ParagraphChangedEventArgs](/javascript/api/word/word.paragraphchangedeventargs) | [**Document**](/javascript/api/word/word.document#word-word-document-onparagraphchanged-member) |
-| `onParagraphDeleted` | Occurs when the user deletes paragraphs.<br><br>Event data object:<br>[ParagraphDeletedEventArgs](/javascript/api/word/word.paragraphdeletedeventargs) | [**Document**](/javascript/api/word/word.document#word-word-document-onparagraphdeleted-member) |
-| `onSelectionChanged` | Occurs when selection within the content control is changed.<br><br>Event data object:<br>[ContentControlSelectionChangedEventArgs](/javascript/api/word/word.contentcontrolselectionchangedeventargs) | [**ContentControl**](/javascript/api/word/word.contentcontrol#word-word-contentcontrol-onselectionchanged-member) |
+When certain changes occur in a Word document, event notifications fire. The Word JavaScript APIs let you register event handlers that allow your add-in to automatically run designated functions when those changes occur. The following events are currently supported.
+
+| Event | Description | Supported objects | Supports coauthoring? |
```

Comment:
Is "Supports coauthoring?" the right framing?  That seems to switch the direction of what triggers the event. Consider something like "Triggered during coauthoring?" to indicate that changes made by other users will trigger events in your session.
```suggestion
| Event | Description | Supported objects | Triggered during coauthoring? |
```